Modules/Microsoft365DSC/DSCResources/MSFT_M365DataAtRestEncryptionPolicy/MSFT_M365DataAtRestEncryptionPolicy.psm1

```diff
@@ -0,0 +1,395 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Identity,
```

Comment:
Is the identity the key parameter here or can we use the `Name` property instead?

Reply:
Identity is human friendly, we can use identity as key.